droid-system-license.inc

```diff
@@ -0,0 +1 @@
+%define system_license ASL 2.0 and BSD and NCSA and zlib
```

Comment:
Do you have some link from where you picked this list? Maybe worth adding here as a comment.

Reply:
Added this comment:
`# grep checked-out AOSP tree for LICENSE files and identify licenses within`